helm/cluster-azure/README.md

```diff
@@ -102,6 +102,8 @@ Properties within the `.metadata` top-level object
 | **Property** | **Description** | **More Details** |
 | :----------- | :-------------- | :--------------- |
 | `metadata.description` | **Cluster description** - User-friendly description of the cluster's purpose.|**Type:** `string`<br/>|
+| `metadata.labels` | **Labels** - These labels are added to the Kubernetes resourses defining this cluster.|**Type:** `object`<br/>|
```

Comment:
I believe this is unrelated to the PR.

Reply:
True. I just updated the docs automatically, and this was added, too.